tools/provision/darwin.sh

```diff
@@ -18,6 +18,9 @@ function main_darwin() {
 
   brew update
 
+  remove_package libressl
```

Comment:
Actually, remove this, and we'll make sure the build hosts have `libre` removed.